components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -672,11 +674,11 @@
                                     <td style="width:30%" style="display:none;">
                                         <input type="hidden" name=<%=value%> id=<%=value%>
```

Comment:
the variable value is iterated in a loop and is defined for each iteration as 
'value = "propertyValue_" + i;'.

Both the string and the integer is defined in the JSP context, and no user input or stored parameter is injected. Hence it is unnecessary.